packages/back-end/src/routers/saved-group/saved-group.controller.ts

```diff
@@ -137,7 +133,7 @@ export const putSavedGroup = async (
   });
 
   // If the values change, we need to invalidate cached feature rules
-  if (savedGroup.values !== values) {
```

Comment:
seems like this was a bug because it was comparing two array using equality which will always return true given that they'll have different references